### DISCUSSION TITLE
Start-up dimensions ?

### DISCUSSION THREAD

**Discussion by ArnaudWald** [Eugeny/tabby#8694] [Q&A] [answered]
Hello, big fan of the app here :heart: 
I have a question though, is it possible to set up the dimensions of the terminal on startup ?

when I start up the app (on Linux), the window is always at the same spot  an takes 3/4th of the screen. I'd like it to be smaller and possibly pushed to a corner 

Thanks for your help !

**Comment by Eugeny** [comment, marked as answer]
Thanks! 

Actually it should remember the window size & position automatically. Can you check whether it gets written into ~/.config/tabby/window.json when you close it?

**Reply by ArnaudWald** [reply]
Hey, thank you for the details about where it's saved, couldn't find it in the in-app settings.

Turns out the dimensions are saved when you properly close the app and not when you turn off your PC with the app still running. So now I know !
